src/structures/size_cabin.jl

```diff
@@ -21,7 +22,7 @@ length.
 function place_cabin_seats(pax, cabin_width, seat_pitch = 30.0*in_to_m, 
     seat_width = 19.0*in_to_m, aisle_halfwidth = 10.0*in_to_m, fuse_offset = 6.0*in_to_m)
 
-    cabin_offset = 10 * ft_to_m #Distance to the front and back of seats
+    cabin_offset = 10 * ft_to_m #Distance to the front of seats
     #TODO the hardcoded 10 ft is not elegant
```

Comment:
agreed. off-the-cuff thought: perhaps it could be a default value for a field within the Cabin struct (should that be fully implemented)